task:
  ros__parameters:
    wait_time: 100 # miliseconds
    global_wp_reached_rad: 3.0 # meters
    start_lat: 63.26480 #-35.3632222
    start_lon: 10.25448 #149.1654373
    finish_lat: 63.26445
    finish_lon: 10.25413
maneuvering:
  ros__parameters:
    distance_to_move: 30.0 # meters
    angle_from_buoys: 20.0 # degrees
    wp_reached_radius: 1.0 #meters
    camera_res_x: 1920
    camera_fov: 70
    wait_time: 5.0 # seconds
    testing_angles: 0
    test_angle: 90.0
collision_avoidance:
  ros__parameters:
    finish_lat: -85.3635673
    finish_lon: 3.1652413
    min_dist: 0.01
    max_dist:  5.0
    fov: 20.0
    obstacle_length: 2.0
    obstacle_length_range: 0.5
    mode_when_obstacles: 0 # 0 for hold, 1 for loiter, 2 for manual

